api/index.js

```diff
@@ -16,17 +16,13 @@ const constants = require('./constants/constants');
 
 // Routes import
 const authRouter = require('./modules/auth/auth.route');
-const movieRouter = require('./modules/movie/movie.route');
```

Comment:
You need to add a license in the beginning of each file
/*
 Copyright (c) 2023, Xgrid Inc, http://xgrid.co

 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at

 http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */

Reply:
Done